diff --git a/setup.py b/setup.py
index 0e4021f..4736d20 100644
--- a/setup.py
+++ b/setup.py
@@ -4,6 +4,7 @@
 DOCLINES = __doc__.split("\n")
 
 from setuptools import setup
+import os
 
 ##########################
 VERSION = "0.0.0dev0"
@@ -36,7 +37,7 @@ setup(
     author_email="kyleabeauchamp@gmail.com",
     description=DOCLINES[0],
     long_description="\n".join(DOCLINES[2:]),
-    version=__version__,
+    version=os.environ["PKG_VERSION"],
     license="MIT",
     url="http://github.com/choderalab/openmoltools",
     platforms=["Linux", "Mac OS-X", "Unix"],
